ansible/roles/test/tasks/advanced-reboot.yml

```diff
@@ -68,8 +83,6 @@
 
     - name: Generate public key for ptf host
       shell: ssh-keygen -b 2048 -t rsa -f /root/.ssh/id_rsa -q -N ""
-      args:
-        creates: /root/.ssh/id_rsa
```

Comment:
Why do you need to remove it?

Reply:
I don't need to. It was removed while debugging some ansible failure. will fix it

Reply:
done